.changeset/sixty-chefs-grow.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.claims.v1": patch
+"@wso2is/i18n": patch
+---
```

Comment:
Let's add console and core as well

Reply:
"@wso2is/console": patch
"@wso2is/core": patch